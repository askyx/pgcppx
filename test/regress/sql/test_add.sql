select test_add(1, 2);