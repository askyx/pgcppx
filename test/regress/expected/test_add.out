select test_add(1, 2);
 test_add 
----------
        5
(1 row)

